src/coreclr/vm/comdynamic.cpp

```diff
@@ -910,5 +887,64 @@ void QCALLTYPE COMDynamicWrite::DefineCustomAttribute(QCall::ModuleHandle pModul
         }
     }
 
+    // Debuggable attribute processing
+    if (((strcmp(szNamespace, FRIEND_ASSEMBLY_TYPE_NAMESPACE) == 0) && (strcmp(szName, FRIEND_ASSEMBLY_TYPE_NAME) == 0)) ||
+        ((strcmp(szNamespace, SUBJECT_ASSEMBLY_TYPE_NAMESPACE) == 0) && (strcmp(szName, SUBJECT_ASSEMBLY_TYPE_NAME) == 0)))
+    {
+        Assembly* pAssembly = pModule->GetAssembly();
+        pAssembly->UpdateCachedFriendAssemblyInfo();
+    }
+}
+
+void QCALLTYPE COMDynamicWrite::DefineCustomAttribute(QCall::ModuleHandle pModule, INT32 token, INT32 conTok, LPCBYTE pBlob, INT32 cbBlob, BOOL toDisk)
```

Comment:
`toDisk` argument is always false and can be deleted too.

Reply:
`GetOnDiskEmitter` and related code can be deleted too

Reply:
Good point... all that code is gone now too.